README.md

```diff
@@ -1,5 +1,6 @@
 Conferences
 ===========
+![Build Jekyll Pages](https://github.com/AndroidStudyGroup/conferences/workflows/Build%20Jekyll%20Pages/badge.svg)
```

Comment:
I would prefer this to be down in the "Running locally" section since it's not the most important thing for someone to see in the README